### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by ridaarif98**
This is a capstone project using HTML, CSS, and javascript.

- All HTML pages are responsive according to screen size
- Popup menu for mobile view
- Display speakers info using javascript object
- Integrate linter
- Resolve linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+           <ul class="secondNavList">
+               <li><a href="./about.html">About</a></li>
+               <li>Program</li>
+               <li>Join</li>
+               <li>Sponsor</li>
+               <li>News</li>
+               <li><a href="./index.html">Summit</a></li>
+           </ul>
+          <a href="./index.html"> <img src="./images/images.png" alt="test"> </a>  
+        </nav>
```
Comment (Temmarie) [inline index.html]: - [ ] You have a blank space at the top of the navbar, kindly remove it.
- [ ] Also, space out the logo and nav items, the logo should be to the extreme left.
- [ ] The last nav item should have a border around it, kindly add this.
- [ ] Also, use the correct background colour here, it shouldn't be black, it should be `#272A31`


_See images below_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    <section class="featuredSpeaker">
+        <h1>Featured Speaker</h1>
+        <hr>
+        <div class="speakerDetail" id="dynamicList">
+        </div>
+        <ul class="buttonList">
+            <li><button class="more-btn" id="more-btn" type="button">More</button></li>
+        </ul>
+        
+    </section>
```
Comment (Temmarie) [inline index.html:126]: - [ ] Good job dynamically rendering the speakers here 👍🏾 
- There's a little tweak to make tho: 
You are missing the image shadow decoration on the top left corner. Kindly add this.

_See images_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+             <li>2021 Creative Commons America</li>
+             <li>Some Rights Reserved</li>
+         </ul>
+         <ul class="footerthirdul">
+          <li>2021 Creative Commons America Some Rights Reserved</li>
+          <li>妇女能顶半边天妇女能顶半边天妇女能顶半边天妇女能顶半边天</li>
+      </ul>
+    
+    </div>
+  </footer>
```
Comment (Temmarie) [inline about.html:147]: - [ ] On the About page, the footer has just one dark part.

_See images_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+           <ul class="footerSecondul">
+               <li>2021 Creative Commons America</li>
+               <li>Some Rights Reserved</li>
+           </ul>
+           <ul class="footerthirdul">
+            <li>2021 Creative Commons America Some Rights Reserved</li>
+            <li>妇女能顶半边天妇女能顶半边天妇女能顶半边天妇女能顶半边天</li>
+        </ul>
+    </div>
+    </footer>
```
Comment (Temmarie) [inline index.html:166]: - [ ] KIndly use the correct colour for the lower footer. It shouldn't be white but `#d3d3d3`

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @ridaarif98     ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A
_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

File: about.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+        <p class="eventDetail">
+          W20 is the official G20 engagement group focused on gender equity. 
+          Its primary objective is to ensure that gender considerations are
+          mainstreamed into G20 discussions and translate into the G20 Leaders’ Declaration
+          as policies and commitments that foster gender equality and women’s economic empowerment.
+        </p>
+
+        <p class="furtherDetail"> Please contact us per Email for any further questions
+          about Women Summit 2021!</p>
+          <p class="emailInfo">womensummit2021@america.org</p>
```
Comment (Temmarie) [inline about.html]: - [ ] This page has a horizontal overflow due to this section on mobile. Kindly fix this.

_See image below_

image

File: index.html [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+
+            </ul>
+            <ul class="firstDetail">
+                
+                <li><img src="./images/conversation.svg" alt="Conversation"></li>
+                <li class="headingList"><h1>Forum</h1></li>
+                <li><p> Have the time to share your thoughts
+                    and opinions with experts for
+                    each topic</p></li>
+            </ul>
```
Comment (Temmarie) [inline index.html]: - [ ] Kindly adjust this section on mobile screens to be properly aligned. You also have a horizontal overflow on this screen size.

_See images below_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: index.html (hunk 2 above)
Comment (Temmarie) [reply index.html:126]: - [ ] Good job adding the background shadow to the images, kindly reduce the size of the shadow and the image and make the colour a light grey(`#d3d3d3`).

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @ridaarif98     ! :wave: :wave:

#### You have done really well in implementing the previously required changes.

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
- [ ] You've done an amazing job overall, but it seems while making the required changes, some styles broke in your code. Kindly fix them

_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.css [hunk 7]
```diff
@@ -0,0 +1,627 @@
...
+    display: none;
+  }
+
+  /* .partner .desktopPartner li {
+    display: inline;
+    color: #919191;
+    line-height: 60px;
+    padding-inline: 20px;
+    margin-right: 20px;
+  } */
```
Comment (mikenath223) [inline about.css:486]: - [ ] OPTIONAL- Kindly remove this commented out code-block. If you need to refer to previous versions of the code please use Git History.

File: about.css [hunk 8]
```diff
@@ -0,0 +1,627 @@
...
+  border: 3px solid #f7f7f7;
+  list-style-type: none;
+  display: flex;
+  flex-direction: column;
+  margin: 35px;
+}
+
+.eventTime {
+  /* background: #FFFFFF;
+  border: 3px solid #F7F7F7; */
```
Comment (mikenath223) [inline about.css:130]: - [ ] OPTIONAL - Kindly remove this comment also.

**Pull request merged**